arch/arm/src/arm/arm_undefinedinsn.c

```diff
@@ -42,6 +42,12 @@
 
 void arm_undefinedinsn(uint32_t *regs)
 {
+  /* Save the saved processor context in current_regs where it can be
+   * accessed for register dumps and possibly context switching.
+   */
+
+  up_set_current_regs(regs);
```

Comment:
What is the difference between saving before and after `_alert()`?

Reply:
`syslog` could trigger another crash and we don't have the registers saved. For example the below one we met.

![image](https://github.com/user-attachments/assets/ebaecc3d-0e53-43cf-972d-92a29e249694)